app/commands/submission/analysis/process.rb

```diff
@@ -53,4 +55,12 @@ def data
   rescue StandardError
     {}
   end
+
+  memoize
+  def tags_data
+    res = JSON.parse(tooling_job.execution_output['tags.json'])
+    res.is_a?(Hash) ? res.symbolize_keys : {}
+  rescue StandardError
```

Comment:
Worth pushing a bugsnag here so we know a representer is misbehaving?

Maybe add a guard before L61 to see if `tooling_job.execution_output['tags.json'])` is missing, and if it is, then  silently return `{}` if the file isn't there. Then add a `bugsnag.notify` to this exception block?